engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/organization.factory.js

```diff
@@ -14,7 +14,20 @@ angular.module('Bastion.organizations').factory('Organization',
         return BastionResource('/katello/api/v2/organizations/:id/:action',
             {id: '@id'},
             {
-                update: { method: 'PUT'},
+                get: {
+                    method: 'GET',
+                    transformResponse: function (data) {
+                        // Some fields cannot be PUT back
+                        var org;
+
+                        org = angular.fromJson(data);
+                        delete org.users;
+                        delete org.parent_id;
+                        delete org.parent_name;
+                        return org;
```

Comment:
would you be opposed to building up what we need (and calling Organization.update()) rather than deleting what we don't?